ocs_ci/ocs/resources/pod.py

```diff
@@ -156,6 +161,25 @@ def get_labels(self):
         """
         return self.pod_data.get('metadata').get('labels')
 
+    def get_pvcname(self):
+        """
+        Get pvcname from pod
+
+        Raises:
+            NotFoundError: If resource not found
```

Comment:
that is not the case if I'm reading the code correctly.
Also, because this is called in init method for POD class, we need to ensure this will never fail but will return None if no PVC found

Reply:
removed this comment, will provide the patch soon

Reply:
marking this as resolved.